### DISCUSSION TITLE
Make an image appear less often as other images

### DISCUSSION THREAD

**Discussion by MaxJGruber** [matteobruni/tsparticles#4365] [Q&A] [answered]
Hey,

Is there a way of having an image particles appear less often than other images. 

If I had an image of a cat and an image of a dog, in Particles right now, the amount of each image would be random. But if I wanted to show 80% dogs and 20% cats, is this possible?

Thanks in advance!

**Comment by matteobruni** [comment, marked as answer]
Yes, it's possible. In the image array duplicate the items at your desired frequency.

For example, if you want 80% dogs and 20% cats, create an image array with 4 dogs and 1 cat.

**Comment by MaxJGruber** [comment]
Awesome! Yeah, easy! Thanks a lot! :)
